roles/translator/src/lib/downstream_sv1/downstream.rs

```diff
@@ -110,6 +110,7 @@ impl Downstream {
         host: String,
         difficulty_config: DownstreamDifficultyConfig,
         upstream_difficulty_config: Arc<Mutex<UpstreamDifficultyConfig>>,
+        task_collector: Arc<Mutex<Vec<(AbortHandle, String)>>>,
```

Comment:
```suggestion
        task_collector: Arc<Mutex<Vec<AbortHandle>>>,
```
Any specific reason for adding string here?

Reply:
this commit will be merged via #1001

the answer to this question is the same as here: https://github.com/stratum-mining/stratum/pull/1001#discussion_r1701015868